### ISSUE TITLE
[question] does it support one org mode file generate multiple posts

### ISSUE THREAD

**Issue by showgood** [niklasfasching/go-org#20]
With Ox-Hugo, I can achieve subtree based approach where each level one heading subtree will become a separate post.

If I don't use Ox-Hugo and utilize this plugin, Is it possible to achieve the same? Went through the documentation, didn't find anywhere mentioned this. Maybe it's not possible, but just want to confirm.

**Comment by niklasfasching** [comment]
This is not possible right now. This feature would require hugo to support multiple output files for a single input file - not sure if that's possible. If it is, I don't think there's much blocking it - all the necessary information should be easily accessible.

Will keep it in my todo - maybe I run into something - but no concrete plans on my side (PRs welcome obviously)

**Comment by pprevos** [comment]
The ox-hugo package in Emacs can hold multiple pages in one Org file.

**Comment by niklasfasching** [comment]
> The ox-hugo package in Emacs can hold multiple pages in one Org file.

I expect ox-hugo does this by just generating multiple markdown files from that one Org file (i.e. hugo produces one output file per input file and life is good).

**Comment by pprevos** [comment]
> I expect ox-hugo does this by just generating multiple markdown files from that one Org file (i.e. hugo produces one output file per input file and life is good). As go-org does not have the chance to generate files inside hugo we can't do this.

That is exactly what it does. You can set Hugo [front matter](https://ox-hugo.scripter.co/doc/org-meta-data-to-hugo-front-matter/) per subtree or per file. I started that way, but the single file quickly became unwieldy.

**Comment by niklasfasching** [comment]
Closing this as it's hugo specific (and even in hugo it can't be done [1]). Since I stopped using hugo myself I don't see myself implementing features for it and thus there's no more need to keep this around. Feel free to open this in hugo and linking here - if hugo  ever makes this possible I'm open to doing whatever has to be done on the go-org side.

[1] afaict hugo has no support for generating multiple output files from a single input file.
